mlflow/server/js/src/components/MetricsPlotView.js

```diff
@@ -100,13 +100,11 @@ export class MetricsPlotView extends React.Component {
     );
 
     const sortedMetricKeys = arrayOfHistorySortedByMetricKey.map((history) => history.metricKey);
-    const deselectedCurvesSet = new Set(deselectedCurves);
     const data = runUuids.map((runUuid, i) => ({
       name: Utils.truncateString(runDisplayNames[i], MAX_RUN_NAME_DISPLAY_LENGTH),
       x: sortedMetricKeys,
       y: arrayOfHistorySortedByMetricKey.map((history) => history[runUuid]),
       type: 'bar',
-      visible: deselectedCurvesSet.has(runUuid),
```

Comment:
Note that this was actually computed incorrectly (should be `!deselectedCurvesSet.has(runUuid)`, as in the line chart) - but specifying visibility is unsupported for bar charts anyways in Plotly

Reply:
You may want to try `visible: 'legendonly'`. Here is an [example](https://codepen.io/rzangcodepen/pen/bGdqXqa?editable=true).
```
...(deselectedCurvesSet.has(runUuid) && { visible: 'legendonly' }),
```